programmers/K번째수.js

```diff
@@ -0,0 +1,21 @@
+function solution(array, commands) {
+    
+    const answer = [];
+    
+    let i = 0;
+    let j = 0;
+    let k = 0;
```

Comment:
```suggestion
    let first = 0;
    let last = 0;
    let target = 0;
```
i, j, k 같은 경우는 for문같이 index를 가리킬 때 사용하는 변수명이에요
보통 변수명 같은 경우는 자신이 어떤 역할을 하는지에 따라 지어주는게 좋아요
진짜 변수명 짓는게 제일 어려워요.. 그래도 생각해보는게 많이 도움이 되실꺼에요